packages/common/src/core/types/IHubInitiativeTemplate.ts

```diff
@@ -1,13 +1,19 @@
 import { IHubItemEntity } from "./IHubItemEntity";
-import { IWithLayout, IWithSlug, IWithCatalog } from "../traits";
+import {
+  IWithLayout,
+  IWithSlug,
+  IWithCatalog,
+  IWithViewSettings,
+} from "../traits";
```

Comment:
These import from a "barrel" file (i.e. index.ts) which we should avoid doing at all costs because it can:
1. Potentially introduce cyclical dependencies
2. Cause issues trying to stub fns in tests
3. Negatively impact tree-shaking leading to larger asset bundles that import things from Hub.js

I'm working on a PR that fully-qualifies import paths in all hub-common modules and tests as part of my efforts to reduce cyclical dependencies in Hub.js. I plan to experiment with some ESLint rules to prevent us from importing from barrel files moving forward.

Reply:
Fixed the imports